src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
   * Update `OpenTelemetry.Api` to `1.8.1`.
   ([#1668](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1668))
 
+* Add support for `OTEL_INSTRUMENTATION_HTTP_KNOWN_METHODS` defined in
```

Comment:
no mention of OTEL_INSTRUMENTATION_HTTP_KNOWN_METHODS in the readme : https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNetCore ?

Reply:
This changelog, as is, is not very useful as it is simply pointing a broad section of the sem. conventions repo. and the link the PR is showing the PR where main->contrib repo migration is done.

Reply:
I can be more specific if we speaking about the PR link, and target it directly to the commit: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1707/commits/d5cabfaf42cc1381a1d7bf023f302c593206d7c2

It should just indicate that we have such support, documentation can be provided later.

Reply:
Maybe I am missing something obvious.. In which PR was the support for `OTEL_INSTRUMENTATION_HTTP_KNOWN_METHODS` added?

Reply:
Part of migration from main repository. Commits are linked in my previous comment. Side effect of code unification

Reply:
@Kielek - We should fix that side effect. Not supporting `OTEL_INSTRUMENTATION_HTTP_KNOWN_METHODS` in AspNetCore and HttpClient is intentional.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/5197#issuecomment-1896198589

Reply:
Fixed by 3b2ecc961670c481d75697eb101cb548949f2b21 - it will be great to have it working in the future. Hopefully it will be supported natively.